### PR TITLE
Swap cbapi for cbc_sdk

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code).
- [x] Linter has passed locally and any fixes were made for failures.
- [x] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

CBAPI-2074

## Pull Request Description

Replacing CBAPI with the CBC SDK

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Existing tests pass

### CODE REVIEW

**Review by abowersox-cb — APPROVED** [review]
Looks like a pretty mechanical change.

File: src/cbc_binary_toolkit/ubs.py [hunk 1]
```diff
@@ -213,12 +213,12 @@ def download_hashes(cbth, hashes, expiration_seconds=3600):
...
 
 
-def get_metadata(cbth, binary):
+def get_metadata(cbc_api, binary):
     """
     Initiates download of binary metadata from Unified Binary Store.
 
     Args:
-        cbth (cbapi.CbThreatHunterAPI): Carbon Black ThreatHunter object.
+        cbc_api (cbc_sdk.CBCloudAPI): Carbon Black ThreatHunter object.
```
Comment (llyon-cb) [inline src/cbc_binary_toolkit/ubs.py:221]: We'll wanna change this like the others to "Carbon Black Cloud API object"

File: src/cbc_binary_toolkit/ubs.py [hunk 2]
```diff
@@ -137,12 +137,12 @@ def _download_binary_metadata(cbth, found_binary):
...
 
-def _validate_download(cbth, download, expiration_seconds):
+def _validate_download(cbc_api, download, expiration_seconds):
     """
     Verifies the presence of Downloads.FoundItem. Retries downloading if there are errors during download.
 
     Args:
-        cbth (CbThreatHunterAPI): Carbon BlackThreatHunter object.
+        cbc_api (cbc_sdk.CBCloudAPI): Carbon Black Cloud API object.
         download (ThreatHunter.Downloads): May contain found, not_found, and error attributes.
```
Comment (llyon-cb) [inline src/cbc_binary_toolkit/ubs.py:146]: This should probably be "download (enterprise_edr.Downloads): "

**Review by llyon-cb — APPROVED** [review]
Just a couple changes to ThreatHunter references, otherwise looks good!

File: src/cbc_binary_toolkit/ubs.py (hunk 1 above)
Comment (avanbrunt-cb) [reply src/cbc_binary_toolkit/ubs.py:221]: Thanks! I thought I caught most but still have a little more to do

File: src/cbc_binary_toolkit/ubs.py (hunk 2 above)
Comment (avanbrunt-cb) [reply src/cbc_binary_toolkit/ubs.py:146]: Thanks! I thought I caught most but still have a little more to do

**Pull request merged**